earth_enterprise/src/fusion/autoingest/sysman/InsetInfo.cpp

```diff
@@ -370,6 +371,25 @@ void FindNeededImageryInsets(
     std::vector<uint>            &neededIndexes,
     uint beginMinifyLevel,
     uint endMinifyLevel) {
+
+  std::vector<khExtents<uint32> > extentsVec;
```

Comment:
This could explain some of the missing performance improvement - it's creating a duplicate vector here. That can be fixed in a real implementation.

Reply:
The `FindNeededImageryInsets` functions were modified to make testing easier in the last investigation. I didn't change it back here, but the prior version could be used. I think that would eliminate this issue.